Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5000,6 +5000,8 @@ SecureContextChecksEnabled:
       default: true
```

Comment:
Hm... there is still webcoreBinding: DeprecatedGlobalSettings above?

Reply:
Fixed, oops. Good catch!